entry/src/main/resources/base/layout/ability_main.xml

```diff
@@ -0,0 +1,478 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
So many Hardcoded strings in this XML, should use `$string: ` resource

Reply:
Move hardcoded string to string.json resource.